unit/unitTestsLensCommon.ml

```diff
@@ -10,7 +10,7 @@ open LensSetOperations
 
 let display_table_query_opt = Conf.make_bool "display_table_query" false "Show queries to take and manipulate tables."
 let leave_tables_opt = Conf.make_bool "leave_tables" false "Do not delete tables after run." 
-let db_host_opt = Conf.make_string "db_host" "localhost" "Database server hostname." 
+let database_args_opt = Conf.make_string "database_args_host" "links:localhost:5432::links" "Database connection args." 
```

Comment:
This looks suspicious to me. You should not hard code database connection information in the test, rather, you should read it from the provided configuration file.

Reply:
So technically this is only the default config. This sets up an argument within the unit testing framework so that if you run `unitTests.exe -help` it will give you all possible command line options related to unit testing including:

`  -database-args-host str           Database connection args. (default: localhost:5432::links)`

Maybe it is worth discussing in the links meeting if we want to use the links config for unit testing, but I personally find it better for the unit testing things to rely on as little infrastructure as possible.

Reply:
It is plain and simple bad software engineering practice. The database configuration differs from system to system. For example, I am running different setups on my server and my workstation. I should not have to edit the same information in two different configuration files to run the test suite.

Reply:
There are already two different configs for testing (`tests/shredding/config.sample` and `tests/database/config.sample`), but I have now just made it fallback to a value in a settings file if there is one that can be loaded.